tests/integration/test_s3_cluster/test.py

```diff
@@ -32,8 +32,9 @@ def create_buckets_s3(cluster):
             # a String, b UInt64
             data = []
 
-            for number in range(100):
-                data.append([str(number) * 10, number])
+            # Make all files a bit different
```

Comment:
why?

Reply:
My hypothesis: 
We have 100 identical files with the same data. During parallel insert select from s3Cluster each replica inserts only 50/100 files (for example first inserts from 1-50 files, second from 51-100). And when insert is finished, both replicas contains identical 5000 rows and for some reason we think that that's all and we don't need to replicate this data between replicas to complete it to 10000 rows (maybe we check checksums of data parts and see that both replicas have the same set of parts). I am not very familiar with replication process, but this looks like the reason (after my change it's not flaky locally)

Reply:
Making 99 files instead of 100 files also helps locally

Reply:
Good thing we have the flaky check, the failure occurs relatively often in CI so it should be a good indication that it's okay now.

Reply:
Just got a confirmation that it's block deduplication process.